strimzi/README.md

```diff
@@ -0,0 +1,237 @@
+# Agent Check: Strimzi
+
+## Overview
+
+This check monitors [Strimzi][1] through the Datadog Agent.
+
+## Setup
+
+### Installation
+
+The Strimzi check is included in the [Datadog Agent][2] package.
+No additional installation is needed on your server.
+
+### Configuration
+
+The Strimzi check collects Prometheus-formatted metrics on the following operators:
+   - Cluster
+   - Topic
+   - User
+
+Follow the instructions below to enable and configure this check for an Agent. This check will only collect metrics from the above listed operators. For monitoring Kafka and Zookeeper, please use the [Kafka][11] and [Zookeeper][12] checks respectively.
+
+#### Host
+
```

Comment:
If this requirement applies to the containerized configuration, it should be moved up to the ##Configuration section.
```suggestion
<div class="alert alert-warning">This check uses <a href="https://docs.datadoghq.com/integrations/openmetrics/">OpenMetrics</a>, which requres Python 3.</div> 

```